android/build.gradle

```diff
@@ -5,6 +5,7 @@ apply plugin: 'kotlin-android-extensions'
 group = "com.github.socure-inc"
 
 buildscript {
+  ext.kotlin_version = '1.7.20'
```

Comment:
Locking to specific versions especially to bug fix versions is not recommended for plugins, most of these things in the gradle file could be changed here so that you do not encounter this in the future (I had changed some previously as well to find versions that would work). Now, in a [typical android project, you would lock your versions](https://developer.android.com/studio/build/dependencies) much like we do on actual app projects but libraries tend to support a much larger amount of versions and having to keep them in parity and release it continually because a version changed is not ideal.

In this case, it would be best to implement [dynamic versions](https://docs.gradle.org/current/userguide/dynamic_versions.html). For instance, kotlin might be best specified as `:1.+` likewise the same for the other classpaths. It can be a trade off many times between potential future BC breaks in a minor release but they're less common.